### DISCUSSION TITLE
Is it possible to hide the blog date URL

### DISCUSSION THREAD

**Discussion by mrbarua** [squidfunk/mkdocs-material#7346] [Q&A] [answered]
Is it possible to hide the blog date URL: /2024/06/15/

Actual URL : 
https://mywebsite.com/weblog/blog/2024/06/15/how-to-change-hostname-in-linux/

I would like to : 
https://mywebsite.com/weblog/blog/how-to-change-hostname-in-linux/

**Comment by alexvoss** [comment, marked as answer]
You can change [URL format for the posts](https://squidfunk.github.io/mkdocs-material/plugins/blog/#config.post_url_format):

```yaml
plugins:
  - blog:
      post_url_format: "{slug}"
```
